bam_plcmd.c

```diff
@@ -84,15 +86,22 @@ static inline void pileup_seq(FILE *fp, const bam_pileup1_t *p, int pos, int ref
         }
         putc(c, fp);
     } else putc(p->is_refskip? (bam_is_rev(p->b)? '<' : '>') : ((bam_is_rev(p->b) && rev_del) ? '#' : '*'), fp);
+    int del_len = -p->indel;
     if (p->indel > 0) {
-        putc('+', fp); printw(p->indel, fp);
-        for (j = 1; j <= p->indel; ++j) {
-            int c = seq_nt16_str[bam_seqi(bam_get_seq(p->b), p->qpos + j)];
-            putc(bam_is_rev(p->b)? tolower(c) : toupper(c), fp);
+        int len = bam_plp_insertion(p, ks, &del_len);
```

Comment:
A check (or assertion) for allocation errors inside `bam_plp_insertion` could be added here.